docs/sources/features/datasources/cloudwatch.md

```diff
@@ -146,35 +183,36 @@ and `dimension keys/values`.
+| _dimension_keys(namespace)_                                             | Returns a list of dimension keys in the namespace.                                                                                                                                 |
+| _dimension_values(region, namespace, metric, dimension_key, [filters])_ | Returns a list of dimension values matching the specified `region`, `namespace`, `metric`, `dimension_key` or you can use dimension `filters` to get more specific result as well. |
+| _ebs_volume_ids(region, instance_id)_                                   | Returns a list of volume ids matching the specified `region`, `instance_id`.                                                                                                       |
+| _ec2_instance_attribute(region, attribute_name, filters)_               | Returns a list of attributes matching the specified `region`, `attribute_name`, `filters`.                                                                                         |
+| _resource_arns(region, resource_type, tags)_                            | Returns a list of ARNs matching the specified `region`, `resource_type` and `tags`.                                                                                                |
+| _statistics()_                                                          | Returns a list of all the standard statistics                                                                                                                                      |
 
 For details about the metrics CloudWatch provides, please refer to the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/CW_Support_For_AWS.html).
 
 #### Examples templated Queries
 
 Example dimension queries which will return list of resources for individual AWS Services:
 
-Query | Service
-------- | -----
-*dimension_values(us-east-1,AWS/ELB,RequestCount,LoadBalancerName)* | ELB
-*dimension_values(us-east-1,AWS/ElastiCache,CPUUtilization,CacheClusterId)* | ElastiCache
-*dimension_values(us-east-1,AWS/Redshift,CPUUtilization,ClusterIdentifier)* | RedShift
-*dimension_values(us-east-1,AWS/RDS,CPUUtilization,DBInstanceIdentifier)* | RDS
-*dimension_values(us-east-1,AWS/S3,BucketSizeBytes,BucketName)* | S3
-*dimension_values(us-east-1,CWAgent,disk_used_percent,device,{"InstanceId":"$instance_id"})* | CloudWatch Agent
-*resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})* | ELB
-*resource_arns(eu-west-1,ec2:instance,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})* | EC2
+| Query                                                                                                                         | Service          |
```

Comment:
Formatting is also messed up here:

![image](https://user-images.githubusercontent.com/434655/69422792-e7f81380-0d24-11ea-9910-344606876fa8.png)